src/components/ArchiveNote.jsx

```diff
@@ -0,0 +1,58 @@
+import axios from "axios"
+import { useNote } from "../contexts/NoteContext"
+import Toast from "./Toast"
+
+export default function ({note}) {
+
+  const {trash, setTrash, setNotes, setArchives} = useNote()
+
+const restoreFromArchive = async (note) =>{
+  try {
+    const response = await axios.post(`/api/archives/restore/${note._id}`, {}, {
+      headers: {
+        authorization: localStorage.getItem("userToken")
```

Comment:
Suggestion: store the token in context then you would not have to get it again n again from local